www/package.json

```diff
@@ -59,11 +59,14 @@
     "@types/chart.js": "^2.7.42",
     "chart.js": "^2.7.3",
     "jquery": "^3.3.1",
+    "jquery-ui": "^1.12.1",
```

Comment:
YOUNG MAN, EXPLAIN YOURSELF! 
![SIN](https://media.giphy.com/media/1Bh3lhR664JVA4pmjH/giphy.gif)

Reply:
Forgive me Father, for I have imported jQuery UI.
I pray that you guide to forgiveness and move past this dark spot in my life.
In the name of the Father, Son, and Holy Spirit. Amen.